mindmeld/query_cache.py

```diff
@@ -126,6 +131,24 @@ def compatible_version(self):
         except Exception:  # pylint: disable=broad-except
             return False
 
+    def compatible_text_pipeline_config(self):
+        """
+        Checks to see if the cache db file exists and that the text preparation pipeline config
+        matches the current pipeline config.
+        """
+
+        cursor = self.disk_connection.cursor()
+        try:
+            cursor.execute("""
+            SELECT version_number FROM version WHERE text_pipeline_hash=(?);
+            """, (self.text_pipeline_hash,))
+            if len(cursor.fetchall()) == 0:
+                # version does not match
+                return False
+            return True
+        except Exception:  # pylint: disable=broad-except
```

Comment:
We should try to stay away from broad exceptions when possible as they can make debugging unseen edge cases difficult in the future

Reply:
Fixed to sqlite3.Error to catch only sqlite DB related exceptions